### ISSUE TITLE
Window Management child windows are wrong size until text changes

### ISSUE THREAD

**Issue by nyanpasu64** [nativeshell/examples#8]
In the Window Management tab, "Show Modal Dialog" and "Show Other Window" spawns windows which are the wrong size:

![Screenshot_20210617_130312](https://user-images.githubusercontent.com/913957/122466281-f690c600-cf6d-11eb-9936-e2f25712754c.png)

![Screenshot_20210617_131520](https://user-images.githubusercontent.com/913957/122466371-13c59480-cf6e-11eb-93e0-377fda6ef2ce.png)

If I click "Show Modal Dialog", then tabbing over to "Show more options..." (which is invisible) and pressing Space or Enter fixes the dialog (and it remains the correct size after I "Hide more options..."):

![Screenshot_20210617_131656](https://user-images.githubusercontent.com/913957/122466536-4c656e00-cf6e-11eb-872d-3897dcc693c5.png)

If I click "Show Other Window", then clicking "Call Method" fixes the window size:

![Screenshot_20210617_131530](https://user-images.githubusercontent.com/913957/122466385-1922df00-cf6e-11eb-9957-2373101a66cc.png)

**Versions:**

I'm running this repo, latest main d385acefcc2405ddb863f79e3ccdb4479891008c, on Linux.

Flutter 2.3.0-17.0.pre.386 • channel master • https://github.com/flutter/flutter.git
Framework • revision c6cd8a60c3 (17 hours ago) • 2021-06-16 22:14:02 -0400
Engine • revision df3aa14215
Tools • Dart 2.14.0 (build 2.14.0-216.0.dev)

**System info:**

Operating System: Arch Linux
KDE Plasma Version: 5.22.1
KDE Frameworks Version: 5.83.0
Qt Version: 5.15.2
Kernel Version: 5.12.10-zen1-1-zen (64-bit)
Graphics Platform: X11
Processors: 12 × AMD Ryzen 5 5600X 6-Core Processor
Memory: 15.6 GiB of RAM
Graphics Processor: NVIDIA GeForce GT 730/PCIe/SSE2

**Comment by knopp** [comment]
This looks like a possible Gtk+ issue.  I haven't tried this with KDE/Plasma yet to be honest.

**Comment by knopp** [comment]
This is actually related to #7.

**Comment by nyanpasu64** [comment]
I think #7 is resolved but this issue persists.

**Comment by knopp** [comment]
This issue should be fixed by [this commit](https://github.com/nativeshell/nativeshell/commit/ef4f640af55785b8992ac0ace64001690c6ea5c0). However neither rust crate or dart package has been updated yet.

**Comment by nyanpasu64** [comment]
Thank you for working on this project! Do you want me to report more bugs and crashes in the example app, nativeshell, and/or Flutter itself, or is this too much work?

**Comment by knopp** [comment]
Please keep reporting them :)

**Comment by nyanpasu64** [comment]
I tried testing the latest nativeshell git, but the dialog sizing issue is not fixed.

I edited Cargo.toml:

```
nativeshell = { git = "https://github.com/nativeshell/nativeshell.git", branch = "main" }
```

Cargo.lock points to:

```
source = "git+https://github.com/nativeshell/nativeshell.git?branch=main#8bfab24602be2eb1bf55a094a8bd15954fa90d8f"
```

The commit, https://github.com/nativeshell/nativeshell/commit/8bfab24602be2eb1bf55a094a8bd15954fa90d8f, is a descendant of your bugfix in https://github.com/nativeshell/nativeshell/commit/ef4f640af55785b8992ac0ace64001690c6ea5c0, but the problem persists.

Also the bug happens on both KDE and GNOME Xorg.
